Azure/azure-cli
# [RDBMS] `az mysql flexible-server create/update`: Fix the logic of `--public-access` parameter

**Related command**

We fix the logic of parameter **public-access** for the server create/update command. This parameter is enum. 
- az mysql flexible-server create/update --public-access [Enabled/Disabled]

**Description**

This PR uses latest Python SDK package which in turn uses the latest GA API version 2022-09-30. This PR will allow customers to enable/disabled public access when creating and updating servers.

**Testing Guide**

See examples in _helptext_mysql.py.

enable/disabled public access when creating and updating servers.

**Testing Guide**

See examples in _helptext_mysql.py.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
